distribution-client-proxy/src/main/java/ca/bc/gov/ag/proxy/DocumentDistributionRequestBuilder.java

```diff
@@ -315,7 +315,7 @@ public DocumentDistributionRequestBuilder setFileNumber(String fileNumber) {
     }
 
     public DocumentDistributionRequestBuilder setDocumentStatus(String documentStatus) {
-        if (documentStatus != null)
+        if (documentStatus != null && !documentStatus.trim().isEmpty())
```

Comment:
Or simply 
```suggestion
        if (!StringUtils.isBlank(documentStatus))
```
StringUtils is available in the apache commons library